src/MoBi.Presentation/Tasks/PKSimStarter.cs

```diff
@@ -86,6 +105,11 @@ private MethodInfo getExpressionCreatorMethod(string methodName)
          return _externalAssembly.GetType(PKSIM_UI_STARTER_EXPRESSION_PROFILE_CREATOR).GetMethod(methodName);
       }
 
+      private MethodInfo getIndividualCreatorMethod(string methodName)
+      {
+         return _externalAssembly.GetType(PKSIM_UI_STARTER_INDIVIDUAL_CREATOR).GetMethod(methodName);
```

Comment:
Is there a reason to have one starter for each? Cannot we have one single entry point that stats the right stuff based on the method? Because if feels like we have one class with one method for each. Soemhow, there is one too many

Reply:
I agree. I suggest we do this refactoring in a new PR. I have created a separate issue for this:

https://github.com/Open-Systems-Pharmacology/MoBi/issues/882

since for the issue I am currently worrking on ( https://github.com/Open-Systems-Pharmacology/MoBi/issues/839 ) we will need to make the class even more extendable - to be able to pass furtherr arguments than just the IShell fe